.buildkite/pipeline.yml

```diff
@@ -24,4 +24,5 @@ steps:
 
   - label: ':hammer: tests'
     commands:
-      - "make test"
+      - "make"
```

Comment:
~How come you added this?~

Edit once I read farther: tests require a firectl binary.  Can you add a comment indicating that you're building the binary to use in the tests?